src/main/java/cubicchunks/asm/mixin/core/client/MixinDebugRenderChunkBorder.java

```diff
@@ -0,0 +1,163 @@
+            bufferbuilder.pos(chunkX + (double)k, minY, chunkZ).color(1.0F, 1.0F, 0.0F, 1.0F).endVertex();
+            bufferbuilder.pos(chunkX + (double)k, maxY, chunkZ).color(1.0F, 1.0F, 0.0F, 1.0F).endVertex();
+            bufferbuilder.pos(chunkX + (double)k, maxY, chunkZ).color(1.0F, 1.0F, 0.0F, 0.0F).endVertex();
+            bufferbuilder.pos(chunkX + (double)k, minY, chunkZ + 16.0D).color(1.0F, 1.0F, 0.0F, 0.0F).endVertex();
+            bufferbuilder.pos(chunkX + (double)k, minY, chunkZ + 16.0D).color(1.0F, 1.0F, 0.0F, 1.0F).endVertex();
+            bufferbuilder.pos(chunkX + (double)k, maxY, chunkZ + 16.0D).color(1.0F, 1.0F, 0.0F, 1.0F).endVertex();
+            bufferbuilder.pos(chunkX + (double)k, maxY, chunkZ + 16.0D).color(1.0F, 1.0F, 0.0F, 0.0F).endVertex();
+        }
+
+        // South-North yellow vertical lines
+        for (int l = 2; l < 16; l += 2)
+        {
+            bufferbuilder.pos(chunkX, minY, chunkZ + (double)l).color(1.0F, 1.0F, 0.0F, 0.0F).endVertex();
+            bufferbuilder.pos(chunkX, minY, chunkZ + (double)l).color(1.0F, 1.0F, 0.0F, 1.0F).endVertex();
+            bufferbuilder.pos(chunkX, maxY, chunkZ + (double)l).color(1.0F, 1.0F, 0.0F, 1.0F).endVertex();
+            bufferbuilder.pos(chunkX, maxY, chunkZ + (double)l).color(1.0F, 1.0F, 0.0F, 0.0F).endVertex();
+            bufferbuilder.pos(chunkX + 16.0D, minY, chunkZ + (double)l).color(1.0F, 1.0F, 0.0F, 0.0F).endVertex();
+            bufferbuilder.pos(chunkX + 16.0D, minY, chunkZ + (double)l).color(1.0F, 1.0F, 0.0F, 1.0F).endVertex();
+            bufferbuilder.pos(chunkX + 16.0D, maxY, chunkZ + (double)l).color(1.0F, 1.0F, 0.0F, 1.0F).endVertex();
+            bufferbuilder.pos(chunkX + 16.0D, maxY, chunkZ + (double)l).color(1.0F, 1.0F, 0.0F, 0.0F).endVertex();
+        }
+
+        // Yellow horizontal lines
+        //start from the offset
+        for (int i1 = (int)yOffset; i1 <= 256+(int)yOffset; i1 += 2)
+        {
+            double d7 = (double)i1 - playerY;
+            bufferbuilder.pos(chunkX, d7, chunkZ).color(1.0F, 1.0F, 0.0F, 0.0F).endVertex();
+            bufferbuilder.pos(chunkX, d7, chunkZ).color(1.0F, 1.0F, 0.0F, 1.0F).endVertex();
+            bufferbuilder.pos(chunkX, d7, chunkZ + 16.0D).color(1.0F, 1.0F, 0.0F, 1.0F).endVertex();
+            bufferbuilder.pos(chunkX + 16.0D, d7, chunkZ + 16.0D).color(1.0F, 1.0F, 0.0F, 1.0F).endVertex();
+            bufferbuilder.pos(chunkX + 16.0D, d7, chunkZ).color(1.0F, 1.0F, 0.0F, 1.0F).endVertex();
+            bufferbuilder.pos(chunkX, d7, chunkZ).color(1.0F, 1.0F, 0.0F, 1.0F).endVertex();
+            bufferbuilder.pos(chunkX, d7, chunkZ).color(1.0F, 1.0F, 0.0F, 0.0F).endVertex();
+        }
+
+        tessellator.draw();
+        GlStateManager.glLineWidth(2.0F);
+        bufferbuilder.begin(3, DefaultVertexFormats.POSITION_COLOR);
+
+        // Blue vertical lines
+        for (int j1 = 0; j1 <= 16; j1 += 16)
+        {
+            for (int l1 = 0; l1 <= 16; l1 += 16)
```

Comment:
More variable names. And formatting. Use the supplied IDEA or eclipse formatter.